modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java

```diff
@@ -85,6 +85,32 @@ public void testGetDDMFormCustomField() {
 		Assert.assertEquals("custom_field", ddmFormField.getType());
 	}
 
+	@Test
+	public void testGetDDMFormFieldRequiredErrorMessage() {
```

Comment:
Hey @mateussgn ,

What do you think to just check the new property in the existing method testGetDDMFormTextField? It already checks other similar properties like confirmationErrorMessage, confirmationLabel and so on.

Reply:
Hey @guilhermedcamacho, I've already made the requested changes and pushed them up, thanks for advising me! 😃